### DISCUSSION TITLE
recoverMessageAddress not accessible

### DISCUSSION THREAD

**Discussion by dylviz** [wevm/viem#217] [Question] [answered]
Hi,

Trying out viem and has been nice so far but ran into an issue where I need to verify a signed message. The docs recommend using recoverMessageAddress:
https://viem.sh/docs/utilities/recoverMessageAddress.html

And I can find it in the repo here:
https://github.com/wagmi-dev/viem/blob/main/src/utils/signature/recoverMessageAddress.ts

But for some reason it doesn't appear to be an exported function in my node_modules directory and my error shows "Cannot find name 'recoverMessageAddress'." which was used as follows:
![Screenshot from 2023-03-14 18-12-42](https://user-images.githubusercontent.com/25091990/225179452-d97f795e-49c6-41cc-89bb-c620a6fe6392.png)

Been looking at this for a little too long. Any input is appreciated. Thanks

**Comment by jxom** [comment, marked as answer]
Nice catch, we were missing the exports. Added them in [here](https://github.com/wagmi-dev/viem/commit/2efa740fea3325e9d80d5552a50fcbb7a0636b71) and released in `0.1.19`.
